store/tikv/snapshot.go

```diff
@@ -49,13 +49,15 @@ var (
 
 // tikvSnapshot implements the kv.Snapshot interface.
 type tikvSnapshot struct {
-	store        *tikvStore
-	version      kv.Version
-	priority     pb.CommandPri
-	notFillCache bool
-	syncLog      bool
-	keyOnly      bool
-	vars         *kv.Variables
+	store           *tikvStore
+	version         kv.Version
+	priority        pb.CommandPri
+	notFillCache    bool
+	syncLog         bool
+	keyOnly         bool
+	vars            *kv.Variables
+	replicaRead     kv.ReplicaReadType
+	replicaReadSeed uint32
```

Comment:
@sunxiaoguang snapshot's `replicaReadSeed` seems always be `0`

maybe we should set it to let `select * from t where pkey = ?` can be loadbalance between sessions

Reply:
Oops, probably it was lost in merge by mistake. Let me fix it.

Reply:
@lysu fixed, please take a look